src/NzbDrone.Common/Instrumentation/NzbDroneFileTarget.cs

```diff
@@ -1,10 +1,12 @@
-﻿using NLog;
+using System;
+using NLog;
 using NLog.Targets;
 
 namespace NzbDrone.Common.Instrumentation
 {
     public class NzbDroneFileTarget : FileTarget
     {
+        [Obsolete]
```

Comment:
Ta/Q I assume instead of putting Obsolete in to fix the error we need to replace this with the new method (if it exists)?

Reply:
ReturnRenderMessage is the replacement

Reply:
Yea, best to just fix it now